doc/source/user-storage.md

```diff
@@ -62,6 +62,17 @@ may be simultaneously attached to a node in your cluster. Check
 your cloud provider for details on the limits of storage
 resources you request.
 
+```eval_rst
+.. note::
+
+   Some cloud providers have a limited amount of allowed storage per node.
```

Comment:
Cloud providers limit the *number* of disks that can be attached per node, regardless of how much storage they have. This limits the number of users that can be present in each node, and usually it is not configurable at all. So the limiting factor is number of users per node, not amount of storage.